tests/molecular/topology_graphs/vertex/placement/cases/metal_complex/fixtures/bidentate.py

```diff
@@ -0,0 +1,111 @@
+import pytest
+import numpy as np
+import stk
+from functools import partial
+from scipy.spatial.distance import euclidean
+
+from ....case_data import CaseData
+
+vertices = stk.metal_complex.vertices
+
+
+@pytest.fixture
+def bidentate(position, building_block_2):
+
+    point1, point2 = points = (
+        position + [-10, 0, 0],
+        position + [-10, 0, 0],
```

Comment:
isnt `point1` and `point2` identical here?

Reply:
Done